boa/src/builtins/object/mod.rs

```diff
@@ -406,6 +407,20 @@ pub fn make_object(_: &mut Value, args: &[Value], ctx: &mut Interpreter) -> Resu
     Ok(object)
 }
 
+/// Uses the SameValue algorithm to check equality of objects
+pub fn is(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
+    let x = args.get(0).cloned().unwrap_or_else(Value::undefined);
+    let y = args.get(1).cloned().unwrap_or_else(Value::undefined);
+
+    if x.get_type() != y.get_type() {
+        return Ok(Value::boolean(false));
+    }
```

Comment:
```suggestion
```
We don't have to check if the values are the same type, because `same_value` checks it https://github.com/boa-dev/boa/blob/a7cffe3c7c/boa/src/builtins/value/equality.rs#L151-L153

Reply:
Sorry, I did not see that you removed it :sweat_smile: